docs(instances): change rescue mode (#1616)
scaleway/docs-content: compute/instances/troubleshooting/change-rescue-mode.mdx

---
meta:
  title: Changing the rescue mode of your Instance

>   description: This page helps you to switch your Instance to the new rescue mode

Reviewer: ```suggestion
  description: This page helps you switch your Instance to the new rescue mode
```

content:
  h1: Changing the rescue mode of your Instance

>   paragraph: This page helps you to switch your Instance to the new rescue mode

Reviewer: ```suggestion
  paragraph: This page helps you switch your Instance to the new rescue mode
```

tags: rescue rescuemode bootscript boot-script instance boot script
dates:
  validation: 2023-05-04
  posted: 2023-05-04
categories:
  - compute
---

> The old, bootscript-based, rescue mode will be phased-out on May 31, 2023. To ensure the correct functioning of your Instances, please make sure to switch them to local boot or the new rescue mode before this date.

Author: The previous Instances rescue mode, which relied on bootscripts, will be gradually discontinued by May 31, 2023. To ensure that your Instances operate correctly, make sure to switch them to either local boot or the new rescue mode before this deadline.

> The previous Instances rescue mode, which relied on bootscripts, will be gradually discontinued by May 31, 2023. To ensure that your Instances operate correctly, make sure to switch them to either local boot or the new rescue mode before this deadline.

Reviewer 2: ```suggestion
The previous Instances rescue mode, which relied on bootscripts, will be discontinued by May 31, 2023. To ensure that your Instances operate correctly, make sure to switch them to either local boot or the new rescue mode before this deadline.
```

Reviewer 3: "Instance rescue mode" singular

<Macro id="iam-requirements" />

<Message type="requirement">
  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)

>   - You have [an Instance](/compute/instances/how-to/create-an-instance) using the old (bootscript-based) rescue mode

Author: ```suggestion
  - You have [an Instance](/compute/instances/how-to/create-an-instance) using the previous (bootscript-based) rescue mode
```

</Message>

1. Click **Instances** in the **Compute** section of the side menu in the console. The list of your Instances displays.
2. Click the name of the Instance that is using the previous rescue mode. The Instance overview displays.

> 3. Stop the Instance if it is currently running by using the toggle at the right hand top of the Instances overview.

Reviewer 4: ```suggestion
3. Stop the Instance if it is currently running by using the top right toggle of the Instances overview page.
```

> 3. Click the **Advanced Settings** tab.

Reviewer 2: numbering wrong :)

5. Scroll to the **Boot Mode** section and change the boot mode of your Instance to **local boot**.

> 6xxx. Power the Instance on again by using the toggle. 

Author: ```suggestion
6. Power the Instance on again by using the toggle. 
```
